diff --git a/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle b/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle
index d86e4f5..de62634 100644
--- a/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle
+++ b/node_modules/react-native-bluetooth-escpos-printer/android/build.gradle
@@ -1,7 +1,6 @@
 buildscript {
     repositories {
-        jcenter { url "http://jcenter.bintray.com/" }
-        maven {url "http://repo.spring.io/plugins-release/"}
+        maven { url "https://repo1.maven.org/maven2" }
         mavenCentral()
         maven {
             // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
@@ -40,8 +39,7 @@ android {
 }
 
 repositories {
-    jcenter { url "http://jcenter.bintray.com/" }
-    maven {url "http://repo.spring.io/plugins-release/"}
+    maven { url "https://repo1.maven.org/maven2" }
     mavenCentral()
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
@@ -53,7 +51,7 @@ repositories {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
+    implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation 'com.facebook.react:react-native:+'  // From node_modules
        implementation 'androidx.legacy:legacy-support-v4:1.0.0'
     implementation "com.google.zxing:core:3.3.0"
